docs/properties.md

```diff
@@ -304,6 +305,35 @@ For both implicit and explicit values, the format is a number.
 }
 ```
 
+### Font
+The Font type cannot be specified implicitly. For explicit values, there are several possible formats.
+
+An object with family, weight and style fields. Each component is a string.
+
+```json
+    "$properties": {
+        "ExplicitExample": {
+            "family": "rbxasset://fonts/families/LegacyArial.json",
+            "weight": "Regular",
+			"style": "Normal"
+        }
+    }
+```
+
+A Font object with family, weight, and style fields. Each component is a string.
+
+```json
+    "$properties": {
+        "ExplicitExample": {
+            "Font": {
+                "family": "rbxasset://fonts/families/LegacyArial.json",
+                "weight": "Regular",
+				"style": "Normal"
+            }
+        }
+    }
+```
```

Comment:
This example should look like other types that have both an implicit and explicit format

Also make sure to check indentation! Here you indented the `style` field with tabs, while `family` and `font` are both indented with spaces. It looks like we're using only tabs in the other examples on this page, so we should keep it consistent.